pkg/splunk/enterprise/clustermaster_test.go

```diff
@@ -165,14 +165,6 @@ func TestApplyClusterMasterWithSmartstore(t *testing.T) {
 		{MetaName: "*v1.ConfigMap-test-splunk-stack1-clustermaster-smartstore"},
 		{MetaName: "*v1.StatefulSet-test-splunk-stack1-cluster-master"},
 		{MetaName: "*v1.Pod-test-splunk-stack1-cluster-master-0"},
-		{MetaName: "*v1.Secret-test-splunk-test-secret"},
```

Comment:
Was this related to MC too?
```{MetaName: "*v1.Secret-test-splunk-test-secret"}```

Reply:
Yes this was for secret generation of MC